site/content/en/latest/user/quickstart.md

```diff
@@ -47,10 +47,16 @@ consideration when debugging.
 
 ## Testing the Configuration
 
+{{< tabs name="tabs_test_the_configuration" >}}
```

Comment:
imo we need to flip this 
1. With External LoadBalancer Support
2. Without External LoadBalancer Support

cc @eitansuez